server/src/main/java/io/spine/server/BoundedContext.java

```diff
@@ -539,4 +458,117 @@ public String toString() {
         return spec.name()
                    .getValue();
     }
+
+    /**
+     * Provides access to features of the context used internally by the framework.
+     *
+     * @throws SecurityException
+     *         if called from outside the framework
+     */
+    @Internal
+    public InternalAccess internalAccess() {
+        Security.allowOnlyFrameworkServer();
+        return this.internalAccess;
+    }
+
+    /**
+     * Provides access to features of {@link BoundedContext} used internally by the framework.
+     */
+    @Internal
+    public class InternalAccess {
```

Comment:
Please make it `final` and move to the upper level. If needed, I'd rather have some `private` API turn into package-private than have `BoundedContext` more bloated.

Also, this looks like a `RepositoryRegistry`.

Reply:
Yes, surely `final`. But it's not just a repository registry. Methods there are not only related to the repositories. I wanted to minimize the number of internally exposed things in `BoundedContext` itself. Maybe, we'll be able to refine it later. For now it's just easing on the `@Internal` API visible for the user of the framework.